aioz/chain.json

```diff
@@ -26,29 +26,29 @@
   },
   "codebase": {
     "git_repo": "https://github.com/AIOZNetwork/go-aioz",
-    "recommended_version": "v1.2.0",
+    "recommended_version": "v1.4.0",
     "compatible_versions": [
-      "v1.2.0"
+      "v1.4.0"
     ],
     "binaries": {
-      "linux/amd64": "https://archive.aioz.network/aiozd-v1.2.0-linux-amd64.tar.gz",
-      "darwin/amd64": "https://archive.aioz.network/aiozd-v1.2.0-darwin-amd64.tar.gz",
-      "windows/amd64": "https://archive.aioz.network/aiozd-v1.2.0-windows-amd64.zip"
+      "linux/amd64": "https://archive.aioz.network/aiozd-v1.4.0-linux-amd64.tar.gz",
+      "darwin/amd64": "https://archive.aioz.network/aiozd-v1.4.0-darwin-amd64.tar.gz",
+      "windows/amd64": "https://archive.aioz.network/aiozd-v1.4.0-windows-amd64.zip"
     },
     "genesis": {
       "genesis_url": "https://archive.aioz.network/aioz_168-1/genesis.json"
     },
     "versions": [
       {
-        "name": "v1.2.0",
```

Comment:
Instead of deleting v1.2.0 from the version history, could you please add v1.4.0 as a new version object below?